sql-cli/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "astro-sql-cli"
-version = "0.3.0"
+version = "0.3.1a0"
```

Comment:
Isn't it worth having this separately? 🤔

Reply:
I wanted to release from the current branch as it was needed for integration efforts. Will create a separate branch by cutting one from this branch and will keep rebasing it.

Reply:
Here is the other PR: https://github.com/astronomer/astro-sdk/pull/1658